src/platforms/platform.dom.js

```diff
@@ -124,11 +124,11 @@ var supportsEventListenerOptions = (function() {
 // https://github.com/chartjs/Chart.js/issues/4287
 var eventListenerOptions = supportsEventListenerOptions ? {passive: true} : false;
 
-function addEventListener(node, type, listener) {
+function addListener(node, type, listener) {
```

Comment:
Not sure why https://github.com/chartjs/Chart.js/commit/cbb7ff7754a0a1619b77a481f6e715c0975c1e6b is showing up on this PR. Seems the rebase didn't totally work

Reply:
when i has all that failing tests i thought that smth really bad went with merge last time. So, i downloaded lates Chart.js files, replaced with it all my local files and commited to my fork. So, in this case my repo should look like the current Chart.js version. So, all that changes came when i replaced files. I have put back my changes again on float-bat support after. Are thise chanages on this file are not correct?

Reply:
i have replaced all files from current chart.js version. Is this file is not correct?

Reply:
i think because this was commited only 2 days ago thats why i picked it

Reply:
The changes shouldn't appear on your PR. The problem is that your PR is comparing to a version of the code from a few days ago probably instead of to the latest master. You should run `git rebase master` again (and maybe make a backup copy first before doing that)

It's probably really hard to rebase with 82 commits spanning a year on the PR. It's possible you might have an easier time opening a new PR at this point rather than rebasing

Reply:
... or you can squash all your 82 commits in a single one, and rebase it on top of master. Then we can keep working in the same PR with all previous comments.

Reply:
@gwyneblaidd if you need help squashing / rebasing this PR, please join our [Slack #dev channel](https://chartjs-slack.herokuapp.com/). But please don't open a new [PR with these 82 commits](/chartjs/Chart.js/pull/6055), it will be useless.

Reply:
Forget my comment about squashing commits, I didn't realize you already merged many times our `master` into your branch, which makes the operation quite complicated, so it would be easier to start a new PR from scratch.

Reply:
rebased PR - https://github.com/chartjs/Chart.js/pull/6056